flink/v1.12/flink/src/main/java/com/netease/arctic/flink/table/ArcticFileSource.java

```diff
@@ -137,9 +137,12 @@ private TableSchema getProjectedSchema() {
       TableSchema.Builder builder = TableSchema.builder().fields(
           projectedColumns,
           Arrays.stream(projectedFields).mapToObj(i -> fullTypes[i]).toArray(DataType[]::new));
+      if (tableSchema.getWatermarkSpecs().size() > 0) {
+        builder.watermark(tableSchema.getWatermarkSpecs().get(0));
+      }
 
       TableSchema ts = builder.build();
-      LOG.info("TableSchema builder after addPrimaryKey, schema:{}", ts);
+      LOG.info("TableSchema builder in getProjectedSchema function, schema:{}", ts);
```

Comment:
I do not recommend including method names of internal code in logs, as this makes it difficult to directly understand the logs. Can it be changed to `Table format after projection:{}`?